zombienet/0004-webrtc/test-webrtc.toml

```diff
@@ -0,0 +1,33 @@
+[settings]
+enable_tracing = false
+
+[relaychain]
+default_image = "{{ZOMBIENET_INTEGRATION_TEST_IMAGE}}"
+default_command = "substrate"
+
+chain = "gen-db"
+chain_spec_path = "zombienet/0004-webrtc/chain-spec.json"
+
+  [[relaychain.nodes]]
+  name = "alice"
+  validator = false
+  db_snapshot="https://storage.googleapis.com/zombienet-db-snaps/substrate/0001-basic-warp-sync/chains-0bb3f0be2ce41b5615b224215bcc8363aa0416a6.tgz"
+  args = ["--listen-addr /ip4/0.0.0.0/udp/30333/webrtc"]
```

Comment:
Hi @melekes, this will require some changes in zombienet (and a new release). Let me work on that and I will update this pr.

Thanks!